server/src/main/java/org/apache/iotdb/db/qp/executor/QueryProcessExecutor.java

```diff
@@ -459,7 +451,7 @@ private void checkPathExists(MNode node, String fullPath, MeasurementSchema sche
   }
 
   @Override
-  public List<String> getAllPaths(String originPath) throws MetadataException {
+  public List<String> matchPaths(String originPath) throws MetadataException {
```

Comment:
It would be nicer to rename the parameter to "pathPattern".

Reply:
I will keep the variable name.